kie-bom/pom.xml

```diff
@@ -157,6 +157,19 @@
         <classifier>sources</classifier>
       </dependency>
 
+      <!-- KIE Security -->
+      <dependency>
+        <groupId>org.drools</groupId>
```

Comment:
Would it be possible to use groupId `org.kie`? Otherwise I guess we should move the module into drools-bom (to follow  the groupId convention).